docs/architecture/adr/ADR030-aws-service-operator.md

```diff
@@ -0,0 +1,29 @@
+# ADR030: AWS Service Operator
+
+## Status
+
+Pending
+
+## Context
+
+[Amazon announced](https://aws.amazon.com/blogs/opensource/aws-service-operator-kubernetes-available/) an AWS Service Operator for Kubernetes in October 2018.
+There is a need for one incoming GSP user, DCS, to provision queues (potentially SQS ones) for their application to interact with.
```

Comment:
I think, as an ADR, this would read better if we didn't talk about a specific user (in this case DCS) and instead focused on the more general requirement.
Just an opinion, not strongly held.